roles/openshift_manageiq/tasks/main.yaml

```diff
@@ -0,0 +1,74 @@
+---
+
+- name: Install ManageIQ service account templates
+  copy:
+    src: manageiq-serviceaccount
+    dest: /etc/origin/
```

Comment:
copying these files over seems a bit overkill here...  I'd almost rather see us have the templates in a variable and then use the shell module to pass the template in through stdin.

For variables in roles/openshift_manageiq/vars/main.yml:
```
manageiq_cluster_role:
  apiVersion: v1
  kind: ClusterRole
  metadata:
    name: management-infra-admin
  rules:
  - resources:
    - pods/proxy
    verbs:
    - '*'
manageiq_serviceaccount:
  apiVersion: v1
  kind: ServiceAccount
  metadata:
    name: management-admin
```

For creating the resources:
```
- name: Create Service Account
  shell: >
    echo {{ manageieq_service_account | to_json }} |
    {{ openshift.common.client_binary }} create
    -n management-infra
    -f /etc/origin/manageiq-serviceaccount/manageiq-serviceaccount.yaml
  register: osmiq_create_service_account
  failed_when: "'already exists' not in osmiq_create_service_account.stderr and osmiq_create_service_account.rc != 0"
  changed_when: false
```